test/passes/optimize-instructions_fuzz-exec.txt

```diff
@@ -0,0 +1,255 @@
+[fuzz-exec] calling test32
+[LoggingExternalInterface logging nan:0x400000]
+[LoggingExternalInterface logging nan:0x400000]
+[LoggingExternalInterface logging nan:0x400000]
+[LoggingExternalInterface logging nan:0x400000]
+[LoggingExternalInterface logging nan:0x7fff82]
+[LoggingExternalInterface logging -nan:0x7fff82]
+[LoggingExternalInterface logging -nan:0x7fff82]
```

Comment:
It looks like there are some non-standard NaNs still floating around. Would it be worth standardizing these cases, too?

Reply:
Fixed, thanks, those were min and max which were missed. There remains a non-standard NaN which is for copysign which is expected, as it does not propagate NaNs with the normal rules.